addons/fleet/models/fleet_vehicle.py

```diff
@@ -85,7 +85,7 @@ def _get_default_state(self):
     horsepower_tax = fields.Float('Horsepower Taxation', compute='_compute_model_fields', store=True, readonly=False)
     power = fields.Integer('Power', help='Power in kW of the vehicle', compute='_compute_model_fields', store=True, readonly=False)
     co2 = fields.Float('CO2 Emissions', help='CO2 emissions of the vehicle', compute='_compute_model_fields', store=True, readonly=False)
-    co2_standard = fields.Char(compute='_compute_model_fields', store=True, readonly=False)
+    co2_standard = fields.Char('CO2 Standard', compute='_compute_model_fields', store=True, readonly=False)
```

Comment:
```suggestion
    co2_standard = fields.Char('CO₂ Standard', compute='_compute_model_fields', store=True, readonly=False)
```
why not using https://www.compart.com/en/unicode/U+2082? 😉